source/ja/docs/front-matter.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Front Matter
+---
+
+Front Matterは、ファイルの先頭に配置されるYAMLまたはJSONのブロックで、投稿の設定のために使用されます。YAMLの場合は三つのハイフン、JSONの場合は三つのセミコロンで終了します。
+
+**YAML**
+
+``` yaml
+---
+title: Hello World
+date: 2013/7/13 20:46:25
+---
+```
+
+**JSON**
+
+``` json
+"title": "Hello World",
+"date": "2013/7/13 20:46:25"
+;;;
+```
+
+### 設定とデフォルト値
+
+設定 | 説明 | デフォルト
+--- | --- | ---
+`layout` | レイアウト | [`config.default_layout`](/docs/configuration#Writing)
```

Comment:
```suggestion
`layout` | レイアウト | [`config.default_layout`](../docs/configuration#Writing)
```

Reply:
Ah, good eye. I fixed this with https://github.com/hexojs/site/pull/2150/commits/c98827484343da82988acd1fc52f813b30e08ba5

Since the heading text of fragment is changed with translation, I also fixed it and similar mistakes in other places